### PR TITLE
Publish Contributor Story on the Cookie Consent Banner project

### PR DESCRIPTION
**Pull request by itrich**
## 📑 Description
This PR adds a Contributor Story on the Cookie Consent Banner project.

## ✅ Checks
- [X] I have read and accept the [Contributor License Agreement](https://opensource.porsche.com/docs/cla)

### CODE REVIEW

File: content/blog/cookie-consent-banner.mdx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+</ImageText>
+
+<ImageText imageSrc="/assets/blog/ccb/1.png" imageAlt="Porsche Digital Campus" theme="light" left
+    title="You are currently maintaining Porsche's open-source project Cookie Consent Banner on GitHub. How would you explain your project to a five-year-old? What problem does your project solve for Porsche?">
+
+    **Patrick:** I'm sure everybody remembers the time when cookie banners hit the
+    internet. It's not only cumbersome for the visitors of a website but also for
+    the maintainers to implement such a solution. But it makes sense, informing the
+    user's what is going on with their data. So we created a component that is not
```
Comment (pboeder) [inline content/blog/cookie-consent-banner.mdx:43]: But it makes sense, informing the user's what is going on with their data and ask them for permission.

File: content/blog/cookie-consent-banner.mdx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    only informative to the visitors of a website, but also easy to integrate for
+    the product teams. It should be as straightforward as possible to integrate
+    third-party scripts and cookies into a website in a privacy-friendly manner,
+    always respecting the decision of the visitor. That's why the component doesn't
+    do any magical gatekeeping of such scripts, but instead making it very
+    convenient for the developer to get to know the information about the consent of
+    the visitor. The developers should have total control over what's going on the
```
[the remark this answers is not in the document]
Comment (itrich) [reply content/blog/cookie-consent-banner.mdx:50]: Can we rephrase this to `The developers should have total control over what's **happening** on the`? This would avoid this *on on* hiccup 😄

File: content/blog/cookie-consent-banner.mdx (hunk 1 above)
Comment (itrich) [reply content/blog/cookie-consent-banner.mdx:43]: Can we completely delete this sentence?

Comment (pboeder) [reply content/blog/cookie-consent-banner.mdx:43]: yes

File: content/blog/cookie-consent-banner.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (pboeder) [reply content/blog/cookie-consent-banner.mdx:50]: much better. Thanks!

File: content/blog/cookie-consent-banner.mdx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    member of the open-source world and to contribute our ambitious standards
+    back to the community.
+    <br/>
+    **Patrick (System Architect My Porsche):** I'm a huge fan of a great user
+    experience. Not only for the end users of a product, but also for the software
+    engineers creating those. An awesome developer experience leads to an awesome
+    (end-) user experience. That's why I enjoy spending time into defining a
+    meaningful and scalable interface definition. The Cookie Consent Banner is such
+    an example. It's a component that you have to build for legal reasons and
+    probably don't touch quite often after the initial implementation. It shouldn't
```
Comment (switchnollie) [inline content/blog/cookie-consent-banner.mdx:31]: `doesn't` instead of `don't`

File: content/blog/cookie-consent-banner.mdx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    be complicated if you need to adapt it again after some time. The Cookie Consent
```
Comment (switchnollie) [inline content/blog/cookie-consent-banner.mdx:32]: `when` instead of `if`

File: content/blog/cookie-consent-banner.mdx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    Banner is very flexible, but still convenient to integrate.
```
Comment (switchnollie) [inline content/blog/cookie-consent-banner.mdx:33]: `yet` instead of `but still`

File: content/blog/cookie-consent-banner.mdx [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    meaningful and scalable interface definition. The Cookie Consent Banner is such
+    an example. It's a component that you have to build for legal reasons and
+    probably don't touch quite often after the initial implementation. It shouldn't
+    be complicated if you need to adapt it again after some time. The Cookie Consent
+    Banner is very flexible, but still convenient to integrate.  
+
+</ImageText>
+
+<ImageText imageSrc="/assets/blog/ccb/1.png" imageAlt="Collection of websites using the cookie banner" theme="light" left
+    title="You are currently maintaining Porsche's open-source project Cookie Consent Banner on GitHub. How would you explain your project to a five-year-old? What problem does your project solve for Porsche?">
```
Comment (switchnollie) [inline content/blog/cookie-consent-banner.mdx:38]: The answer won't be understandable for a five-year-old. Maybe rephrase the question to `in simple terms`?

File: content/blog/cookie-consent-banner.mdx [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+    **Patrick:** I'm sure everybody remembers the time when cookie banners hit the
```
Comment (switchnollie) [inline content/blog/cookie-consent-banner.mdx:40]: `first appeared on` instead of `hit`

File: content/blog/cookie-consent-banner.mdx [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+<ImageText imageSrc="/assets/blog/ccb/2.jpg" imageAlt="Chart showing weekly downloads of the Cookie Consent Banner" theme="light"
+    title="With over 3000 weekly downloads your project already gained some adoption among the community. What did you learn along the way of maintaining an open-source project?">
+
+    **Patrick:** The component should be easy to use, thus we need to ensure a
+    balance between sensible defaults and a potentially overwhelming set of
+    options to customize. Given the success of the component, I would say we
+    found a good balance. I really like the feedback the community provides via
+    GitHub Issues and Pull Requests. For example, did we face an issue where
+    someone mentioned how the component affects their Google Lighthouse Score.
```
Comment (switchnollie) [inline content/blog/cookie-consent-banner.mdx:67]: Replace phrase with "For example, one of the issues we had was someone mentioning how the component was affecting their Google Lighthouse score."

File: content/blog/cookie-consent-banner.mdx [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+    Together, we found a solution, making the component better performing for
```
Comment (switchnollie) [inline content/blog/cookie-consent-banner.mdx:68]: `performing better` instead of `better performing`

File: content/blog/cookie-consent-banner.mdx (hunk 6 above)
Comment (itrich) [reply content/blog/cookie-consent-banner.mdx:38]: I rephrased it to "in a nutshell"
